rust/rope/src/tree.rs

```diff
@@ -948,4 +968,19 @@ mod test {
         }
     }
 
+    #[test]
```

Comment:
No test for `at_or_prev`?

Reply:
Basic functionality for both is included in this test case, didn't feel this was complicated enough to warrant serious energy.